### PR TITLE
Added contributing and License in main documentation 

### LINKED ISSUE #156: Set a convention for docstrings
I suggest either the [Google Python style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#google-vs-numpy) or the [sphinx convention](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format)

Also see [PEP257 docstring convention](https://www.python.org/dev/peps/pep-0257/). 

Once we agree on a convention, we should update: https://github.com/neuropoly/ivado-medical-imaging/blob/master/CONTRIBUTING.md#developing

### PR DESCRIPTION
**Pull request by jcohenadad**
Changes include:
- Added symlinks for CONTRIBUTING and LICENSE under docs/ to be able to include them in the docs build;
- Added Contributing and License in the documentation TOC;
- Updated CONTRIBUTING.md:
  - Fix a few issues and clarify things
  - Added docstrings convention info (Google style). Fixes #156

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -241,37 +200,32 @@ If the PR fixes issue(s), indicate it after your introduction:
...
-#### Continuous Integration
-
+### Continuous Integration
 
-The PR can\'t be merged if [Travis
-build](https://travis-ci.org/neuropoly/ivado-medical-imaging) hasn\'t
-succeeded. If you are familiar with it, consult the Travis test results
+The PR can't be merged if [Github Actions "Run tests"](https://github.com/neuropoly/ivado-medical-imaging/actions)
+hasn't succeeded. If you are familiar with it, consult the test results
 and check for possibility of allowed failures.
```
Comment (kousu) [inline CONTRIBUTING.md:207]: We should try not to make a habit of allowing failures. Pytest has an escape hatch for this condition: `@pytest.mark.xfail` by default is in `strict=False` mode, i.e. it handles flaky tests. https://docs.pytest.org/en/latest/reference.html#pytest-mark-xfail-ref. It's clearer to use that, or to comment out a test entirely with a comment saying that it's flaky, than needing individuals to remember what tests should fail.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -156,30 +121,30 @@ Make sure to cite any papers, algorithms or articles that can help
...
+-- which you probably care about -- does not stop working in the
+future.
 
-#### Licensing
+### Licensing
 
 Ensure that you are the original author of your changes, and if that is
 not the case, ensure that the borrowed/adapted code is compatible with
-the MIT license.
+the [project's license](https://ivado-medical-imaging.readthedocs.io/en/latest/license.html).
```
Comment (kousu) [inline CONTRIBUTING.md:142]: Could this be a relative link? Being absolute means this repo -- and all forks of it -- and that site are glued together.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,58 +1,56 @@
+# Contributing to ivadomed
 
+## Introduction
 
-# Contributing to IVADO Medical Imaging
+First off, thanks for taking the time to contribute!
```
Comment (kousu) [inline CONTRIBUTING.md:5]: :rocket:

File: CONTRIBUTING.md (hunk 2 above)
Comment (jcohenadad) [reply CONTRIBUTING.md:142]: i would prefer it to be relative indeed, but i didn't find an easy way to redirect to the RTD's TOC entry. The fact that this file is a symlink (and in md format, which is further processed by Recommark for sphinx compatibility) makes it tricky...

File: CONTRIBUTING.md (hunk 1 above)
Comment (jcohenadad) [reply CONTRIBUTING.md]: agreed-- fixed in aedf677c38b6094a2e3e12387977fdc29c0d2e56

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,58 +1,56 @@
...
-    2. [Submitting an Issue](#submitting-an-issue)
-3. [Contributing to IVADO Medical Imaging (Pull request)](#contributing-to-ivado-medical-imaging-pull-request)
-    1. [Opening a Branch](#opening-a-branch)
-    2. [Naming your Branch](#naming-your-branch)
-    3. [Developing](#developing)
-    4. [Committing](#committing)
-    5. [Submitting a Pull Request](#submitting-a-pull-request)
-4. [Versioning](#versioning)
+When contributing to this repository, please first discuss the change you wish
+to make via Github issue, email, or any other method with the core team before
```
Comment (charleygros) [inline CONTRIBUTING.md:8]: email: should we add an email address? alternatively "Github issue" could be enough

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -226,12 +187,10 @@ for each new release, so please follow the following rules:
     Title](#issue-title)).
 -   Do not include the applicable issue number in the title (do it in
     the [PR Body](#pr-body)).
--   Do not include the function name (use a [PR Labels]() instead).
 -   If the PR is not ready for review, add \"(WIP)\" at the beginning of
```
Comment (charleygros) [inline CONTRIBUTING.md:190]: WIP --> "Convert to draft"?

File: CONTRIBUTING.md (hunk 4 above)
Comment (jcohenadad) [reply CONTRIBUTING.md]: indeed-- fixed

File: CONTRIBUTING.md (hunk 5 above)
Comment (jcohenadad) [reply CONTRIBUTING.md]: yup!

**Pull request merged**